tailwind.config.js

```diff
@@ -51,12 +47,6 @@ module.exports = {
           "0 4px 4px var(--theme-color-900)",
         ],
       },
-      height: {
-        xl: "36rem",
-        "60vh": "60vh",
-        "75vh": "75vh",
-        "80vh": "80vh",
-      },
```

Comment:
All of the custom h- classes were unused. These were all artifacts of the non-scrolling layout where everything had to fit inside the screen's dimensions